backend/api/Database/Models/Robot.cs

```diff
@@ -37,8 +37,8 @@ public Robot(CreateRobotQuery createQuery)
             IsarId = createQuery.IsarId;
             Name = createQuery.Name;
             SerialNumber = createQuery.SerialNumber;
-            CurrentInstallation = createQuery.CurrentInstallation;
-            CurrentArea = createQuery.CurrentArea;
+            CurrentInstallation = installation;
+            CurrentArea = area ?? null;
```

Comment:
Does this ?? null prevent a warning? Since I would assume it's equivalent to just having "area" in this case since it's already nullable

Reply:
Thought that it did but it seems that it doesn't, changed it